### PR TITLE
Reset subresource test: check that boot time string is not empty

### PR DESCRIPTION
**Pull request by iholder101**
### What this PR does
A super duper tiny follow-up to https://github.com/kubevirt/kubevirt/pull/14313.
Checks that the boot time string is not empty, so the test won't pass as a false positive. see https://github.com/kubevirt/kubevirt/pull/14313#discussion_r2009129561.

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/compute/subresources/reset.go [hunk 1]
```diff
@@ -60,7 +60,8 @@ var _ = Describe(compute.SIG("Reset subresource", func() {
 			bTimePostReset, err := console.RunCommandAndStoreOutput(vmi, cmd, time.Second*30)
 			Expect(err).ToNot(HaveOccurred())
 
-			By("Check the pre and post reset boot times are different")
+			By("Check the pre and post reset boot times are different and non-empty")
+			Expect(bTimePreReset).ToNot(BeEmpty())
```
Comment (dasionov) [inline tests/compute/subresources/reset.go:64]: how can we be confident the boot times will always differ, and how do we handle cases where they might actually end up being the same?

Comment (iholder101) [reply tests/compute/subresources/reset.go:64]: The test boots up a VM, gathers the boot time, reboots it, then gathers the new boot time.
I don't see how they can end up the equal :)

**Comment by dasionov** [comment]
Thank you! 
/lgtm

File: tests/compute/subresources/reset.go (hunk 1 above)
Comment (dasionov) [reply tests/compute/subresources/reset.go:64]: guess it's very low chance for the timings to hit the same boot time, so at this point we can probably ignore this edge case.

Comment (iholder101) [reply tests/compute/subresources/reset.go:64]: I understand your concern, however the boot time here is represented as a unix timestamp, so the granularity is of seconds. A full reboot, connection to the guest agent and the execution of the command is impractical to take less than a second.

Thanks for raising this!

**Comment by iholder101** [comment]
> We need to check that both pre and post time are not empty
> 
> ```go
> Expect(bTimePreReset).ToNot(BeEmpty())
> Expect(bTimePostReset).ToNot(BeEmpty())
> ```
> 
> Thank you!

I think that's redundant, as we later check that they are not equal.
If one is not empty and the strings aren't equal, the second cannot be also empty.

**Comment by fossedihelm** [comment]
> I think that's redundant, as we later check that they are not equal. If one is not empty and the strings aren't equal, the second cannot be also empty.

Why not? :)

Consider this scenario:
```
bTimePreReset = "something"
bTimePostReset = ""
```

`Expect(bTimePreReset).ToNot(Equal(bTimePostReset))` succeeded, but I don't think this is consistent.

**Comment by iholder101** [comment]
> > I think that's redundant, as we later check that they are not equal. If one is not empty and the strings aren't equal, the second cannot be also empty.
> 
> Why not? :)
> 
> Consider this scenario:
> 
> ```
> bTimePreReset = "something"
> bTimePostReset = ""
> ```
> 
> `Expect(bTimePreReset).ToNot(Equal(bTimePostReset))` succeeded, but I don't think this is consistent.

Damn it, I'm too tired :laughing: :baby: 
you're right, sorry.

**Comment by fossedihelm** [comment]
/lgtm
Thank you!
@jean-edouard PTAL

**Review by jean-edouard — COMMENTED** [review]
/approve
